ons_alpha/jinja2/templates/pages/bulletin_page.html

```diff
@@ -113,24 +114,31 @@ <h1 class="ons-u-fs-3xl bulletin-header__heading">
                 {% endfor %}
 
                 {% if page.contact_details %}
-                    <section id="contact-details">
-                        <h2>{{ _("Contact details") }}</h2>
-
-                        <div>
-                            <h3 class="ons-u-mb-no ons-u-fs-r--b">Name</h3>
-                            <p>{{ page.contact_details.name }}</p>
-                            <h3 class="ons-u-mb-no ons-u-fs-r--b">Email</h3>
-                            <p>
-                                <a href="mailto:{{ page.contact_details.email }}">{{ page.contact_details.email }}</a>
+                    <div class="contact-details">
```

Comment:
https://service-manual.ons.gov.uk/design-system/components/table-of-contents is specific about the markup element it wants for in-page table of contents:

> wrap each section of content in the second column in a `<section>` tag with a unique id. For example: `<section id="section1">`

Reply:
aha! I didn't realise that. I will revert.